internal/metrics/os_disk_stats_helper_test.go

```diff
@@ -7,13 +7,14 @@ import (
 	"github.com/google/uuid"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
 )
 
 var _ = Describe("OS disk stats helper", func() {
 
 	var (
 		tempDir         string
-		diskStatsHelper *OSDiskStatsHelper = NewOSDiskStatsHelper()
+		diskStatsHelper *OSDiskStatsHelper = NewOSDiskStatsHelper(logrus.New())
```

Comment:
consider discarding output from the logger for the tests

Reply:
What do you mean?

Reply:
For example https://github.com/openshift/assisted-service/blob/ecbb8a578309e35728c0bf10ec10227914230aa0/pkg/thread/example_test.go#L19

IIRC we should have a utility function that initializes the logger for tests